src/Components/DataHistoryExplorer/Internal/TimeSeriesViewer/TimeSeriesViewer.tsx

```diff
@@ -0,0 +1,308 @@
+            const iconStyles = { marginRight: '8px' };
+
+            return (
+                <div style={{ display: 'flex' }}>
+                    <Icon
+                        style={iconStyles}
+                        iconName="StackedLineChart"
+                        aria-hidden="true"
+                    />
+                    <span>{option.text}</span>
+                </div>
+            );
+        },
+        [t]
+    );
+    const handleOnRenderAggregationTypeTitle = useCallback(
+        (options: IDropdownOption[]): JSX.Element => {
+            const option = options[0];
+            const iconStyles = { marginRight: '8px' };
+
+            return (
+                <div style={{ display: 'flex' }}>
+                    <Icon
+                        style={iconStyles}
+                        iconName="AssessmentGroupTemplate"
+                        aria-hidden="true"
+                    />
+                    <span>{option.text}</span>
+                </div>
+            );
+        },
+        [t]
+    );
+    const updateXMinAndMax = useCallback(() => {
+        const nowInMillis = Date.now();
+        xMinDateInMillisRef.current =
+            nowInMillis - selectedQuickTimeSpanInMillis;
+        xMaxDateInMillisRef.current = nowInMillis;
+    }, [selectedQuickTimeSpanInMillis]);
+    const highChartSeriesData: Array<IHighChartSeriesData> = useMemo(
+        () =>
+            timeSeriesTwinList.length
+                ? transformADXTimeSeriesToHighChartsSeries(
+                      data,
+                      timeSeriesTwinList
+                  )
+                : [],
+        [data, timeSeriesTwinList]
+    );
+
+    // side effects
+    const prevQuery = usePrevious(query);
+    useEffect(() => {
+        if (query && query !== prevQuery) {
+            fetchTimeSeriesData();
+            updateXMinAndMax();
+        }
+    }, [query]);
+
+    // styles
+    const classNames = getClassNames(styles, {
+        theme: useTheme()
+    });
+
+    return (
+        <div className={classNames.root}>
+            <Pivot
+                overflowBehavior={'menu'}
+                styles={classNames.subComponentStyles.pivot}
+            >
+                <PivotItem
+                    headerText={t('dataHistoryExplorer.viewer.chart')}
+                    itemKey={ViewerPivot.Chart}
+                >
+                    {timeSeriesTwinList.length === 0 ? (
+                        <IllustrationMessage
+                            descriptionText={t(
+                                'dataHistoryExplorer.viewer.noSeriesDescription'
+                            )}
+                            type={'info'}
+                            width={'compact'}
+                            imageProps={{
+                                src: GenericErrorImg,
+                                height: 172
+                            }}
+                            styles={{ container: { flexGrow: 1 } }}
```

Comment:
Please move this to styles file

Reply:
I am going to change this to command bar in the subsequent PR, pending on this change now.